.gitpod.yml

```diff
@@ -0,0 +1,32 @@
+image:
+    file: .gitpod.dockerfile
+ports:
+    - port: 3000
+      onOpen: open-preview
+tasks:
+    - init: >
+        export CHROME_BIN=/usr/bin/chromium-browser &&
+        gem install bundler &&
+        bundle config set without 'production' &&
+        ./lib/exporter-deps.sh > /dev/null 2>&1 &&
+        bundle install &&
```

Comment:
Yay! It gave up after this line but maybe with the newly added `rvm install 2.4.6` it'll make it all the way through?

Reply:
actually it might've given up on line 9 because I don't see gdal installed.

Reply:
No, so `Required ruby-2.4.6 is not installed` still happens. we need to set the ruby version correctly OR to update the gemfile to the version Gitpod has.

Reply:
ok i set it 2.4.6 in the dockerfile.

Reply:
OK, i'm still not sure why the setup steps aren't all running but it's copying the config files now and set to 2.4.6... but it still needs `bundle install` run... weird.

Reply:
Now it doesn't need `bundle install` and line 16 (create test database) is already run. but it seems to not get to line 18 because it says there are pending migrations...

Reply:
But the tests run!

Reply:
yarn install had also not yet run.

So currently we just need to run:

`rake db:setup && yarn install` to get it running completely!

Reply:
AHA -- the last 2 had no && after them. So maybe they were executed asynchronously or something. I've chained them now, let's see.